common/app/model/Asset.scala

```diff
@@ -82,20 +82,23 @@ object VideoAsset {
     VideoAsset(
       fields = Helpers.assetFieldsToMap(asset),
       mimeType = asset.mimeType,
-      url = asset.file )
+      url = asset.file,
+      secureUrl = asset.typeData.get.secureFile )
```

Comment:
Any reason to `get` rather than `flatMap` here?

I presume that in the vast majority of cases this would make no difference as videos would have typeData, but it'd be a bit more consistent with other accesses of this.